app/src/main/java/com/swent/assos/ui/screens/manageAsso/CreateEvent.kt

```diff
@@ -246,11 +268,10 @@ fun CreateEvent(assoId: String, navigationActions: NavigationActions) {
                 Button(
                     enabled =
                         event.description.isNotEmpty() &&
-                            event.image.isNotEmpty() &&
+                            event.image != Uri.EMPTY &&
                             event.title.isNotEmpty() &&
                             event.startTime != null &&
-                            event.endTime != null &&
-                            event.fields.isNotEmpty(),
+                            event.endTime != null,
```

Comment:
I do not think we should add an event if the fields are empty. It would be just an image and a title